rotkehlchen/db/eth2.py

```diff
@@ -584,7 +585,7 @@ def process_accumulating_validators_balances_and_pnl(
                 events.extend(events_db.get_history_events(
                     cursor=cursor,
                     filter_query=filter_query,  # type: ignore[arg-type]  # no overload for EthStakingEventFilterQuery
-                    has_premium=True,
+                    entries_limit=INTERNAL_QUERY_LIMIT,
```

Comment:
here I believe we don't want to change this

Reply:
what do you mean here @yabirgb ?